extensions/lesion-tracker/src/index.js

```diff
@@ -1,12 +1,4 @@
-// SEE: Async imports (code-splitting)
-// import asyncComponent from './asyncComponent.js';
-// import OHIFDicomPDFSopClassHandler from './OHIFDicomPDFSopClassHandler.js';
-
-// const ConnectedOHIFDicomPDFViewer = asyncComponent(() =>
-//   import(
-//     /* webpackChunkName: "ConnectedOHIFDicomPDFViewer" */ './ConnectedOHIFDicomPDFViewer'
-//   )
-// );
+import LesionTrackerPanel from './components/LesionTrackerPanel';
```

Comment:
We do _eventually_ want this to be generic. Can we change this to MeasurementComparisonTable or something similar?

Reply:
Worth noting it's technically not a panel. It's content that we ultimately end up slotting into a panel. The distinction helps note that we could throw this in a modal or some other container if the need arose.

Reply:
Done, thanks.